src/Integration.UnitTests/MefServices/ActiveSolutionBoundTrackerTests.cs

```diff
@@ -74,7 +78,7 @@ public void ActiveSolutionBoundTracker_ArgChecls()
         }
 
         [TestMethod]
-        public void ActiveSolutionBoundTracker_Unbound()
+        public void ActiveSolutionBoundTracker_Initialisation_Unbound()
```

Comment:
Miscellaneous: renamed method to make it clearer what was being tested. Same for next method.